threescale_api/resources.py

```diff
@@ -1465,9 +1465,10 @@ def metrics(self) -> 'BackendMetrics':
     def mapping_rules(self) -> 'BackendMappingRules':
         return BackendMappingRules(parent=self, instance_klass=BackendMappingRule)
 
-    @property
-    def usages(self) -> 'BackendUsages':
-        return BackendUsages(parent=self, instance_klass=BackendUsages)
+    def usages(self) -> list:
+        """ Returns list of backend usages where the backend is used."""
+        return [back_usage for ser in self.threescale_client.services.list()
```

Comment:
I do not fully understand this loop, could you please elaborate more?

Reply:
BackendUsage objects are available via service objects, see `/admin/api/services/{service_id}/backend_usages.json` so we need to iterate all services to select only BackendUsages which belong to Backend represented by `self`. It is not possible to use similar solution as in https://github.com/3scale-qe/3scale-api-python/blob/master/threescale_api/resources.py#L1208